src/main/java/fr/cnes/sonar/report/exporters/xlsx/XlsXTools.java

```diff
@@ -236,7 +238,11 @@ public static void addSelectedData(List<Issue> issues, XSSFSheet selectedSheet,
                 row.createCell(RULE_ID_INDEX).setCellValue(issue.getRule());
                 row.createCell(MESSAGE_INDEX).setCellValue(issue.getMessage());
                 row.createCell(ISSUE_TYPE_INDEX).setCellValue(issue.getType());
-                row.createCell(ISSUE_SEVERITY_INDEX).setCellValue(issue.getSeverity());
+                if(issue.getType().equals(StringManager.HOTSPOT_TYPE)){
+                    row.createCell(ISSUE_SEVERITY_INDEX).setCellValue(StringManager.HOTSPOT_SEVERITY);
```

Comment:
Why ? You already set it DataAdapter

Reply:
DataAdapter did not take into account the change for the xls file.
So I had to do a condition because "issue.getSeverity())" can't find anything for a security hotspot

Reply:
Why ?